src/core/agent/services/multiSigService.ts

```diff
@@ -953,7 +949,7 @@ class MultiSigService extends AgentService {
       .get(ourIdentifier.id as string);
 
     await this.sendMultisigExn(
-      ourIdentifier.signifyName,
+      ourIdentifier.id,
```

Comment:
We should update `sendMultisigExn` and any other functions to have a `prefix` param instead of `name`, otherwise it's confusing. Please update everywhere you see this (where we are really passing the prefix/ID)